CMakeLists.txt

```diff
@@ -484,6 +491,7 @@ IF (NETCDF_C_INCLUDE_DIR)
    string(REGEX MATCH "[01]" USE_NETCDF4 "${macrodef}")
    IF (USE_NETCDF4)
       MESSAGE(STATUS "Whether NetCDF-C built with HDF5 enabled: yes")
+      FIND_PACKAGE(HDF5 COMPONENTS C HL REQUIRED)
```

Comment:
The netcdf-c cmake file "netCDFTargets.cmake" contains two HDF5 aliases.  They do not resolve unless I use FIND_PACKAGE to locate HDF5.